gcloud-java-contrib/gcloud-java-nio/src/test/java/com/google/gcloud/storage/contrib/nio/CloudStorageFileAttributesTest.java

```diff
@@ -29,14 +31,13 @@
 
   private static final byte[] HAPPY = "(✿◕ ‿◕ )ノ".getBytes(UTF_8);
 
-  @Rule
-  public final AppEngineRule appEngineRule = new AppEngineRule();
 
   private Path path;
   private Path dir;
 
   @Before
   public void before() throws Exception {
+    CloudStorageFileSystemProvider.setGCloudOptions(LocalGcsHelper.options());
```

Comment:
> it is valid use-case for the test to fail if that happen

Could you clarify what you mean here?

> the reader/maintainer assumes that some code in that method can throw that exception

I'm not so sure someone would think that, if `throws Exception` is applied uniformly to everything in the test class.

But either way, I don't feel strongly enough to push back any further. I'll remove all the superfluous `throws Exception` cases when I do my auto-format PR after this one.